modules/manage/partials/authentication.adoc

```diff
@@ -1295,23 +1324,52 @@ endif::[]
+- Schema Registry API: Management of schemas and schema evolution
+- HTTP Proxy API: RESTful interface for Kafka clients
 
-=== Basic authentication
+=== Permission models
 
-Basic authentication provides a method for securing HTTP endpoints. With basic authentication enabled, HTTP user agents, such as web browsers, must provide a username and password when making a request.
+Each of the HTTP APIs implements its own permission model with different levels of access control. This section lists what permissions are available to different user types, and how to enable authentication.
 
-Before you can enable basic authentication for these HTTP APIs, you must <<sasl,enable SASL authentication for the Kafka API>>. Then, both SCRAM users and superusers can use their credentials to access them:
+==== Admin API permissions
 
-* *HTTP Proxy*: Access to the Kafka API impersonates the user whose credentials were used to authenticate to HTTP Proxy, and the user is subject to authorization restrictions by Redpanda ACLs. To support this design, Redpanda passes the username/password in memory to a SASL-enabled Kafka client.
-* *Schema Registry*: Authorization is "all or nothing": if the user presents a valid user account, then they have full read/write access.
+The Admin API primarily requires superuser privileges, with a few exceptions for read-only status endpoints.
+
+For a complete list of all Admin API endpoints, see see xref:api:ROOT:admin-api.adoc[].
+
+==== Schema Registry API permissions
+
+The Schema Registry implements a granular permission model.
+
+Most read operations require only user-level authentication, while write operations that affect the global state or mode typically require superuser privileges.
+
+==== HTTP Proxy permissions
+
+The HTTP Proxy API impersonates the authenticated user when making Kafka API calls:
+
+
+Regular users are subject to the same ACL restrictions as they would be when using the Kafka API directly.
+
+=== Enable authentication
+
+Redpanda supports authentication for the HTTP APIs using either basic authentication or OIDC (OpenID Connect).
+
+==== Prerequisites
+
+Before enabling authentication for the HTTP APIs, you must <<sasl,enable SASL authentication for the Kafka API>>. This creates the credential store that HTTP authentication will use.
+
+==== Basic authentication
+
+Basic authentication provides a method for securing HTTP endpoints. With basic authentication enabled, HTTP user agents, such as web browsers, must provide a username and password when making a request.
 
 To add users to the Redpanda credential store that HTTP basic authentication uses, create users with xref:reference:rpk/rpk-acl/rpk-acl-user-create.adoc[`rpk security user create`].
 
+Both SCRAM users and superusers can use their credentials to access them. The HTTP Proxy API and the Schema Registry API use the same credential store as the Kafka API, so you can use the same credentials for all three APIs.
```

Comment:
What is "them" referring to here?
Both SCRAM users and superusers can use their credentials to access **them**.